scripts/release_scripts/update_configs.py

```diff
@@ -45,7 +45,7 @@
 CONSTANTS_REGEX = '^(  "[A-Z_]+": ).*$'
 TERMS_PAGE_URL = (
     'https://github.com/oppia/oppia/commits/develop/core/'
-    'templates/pages/terms-page/terms-page.mainpage.html')
+    'templates/pages/terms-page/terms-page.component.html')
```

Comment:
Optional: Can we have a backend test to make sure this file exists? (Just to avoid such an issue to happen again, I'm also fine as is considering we don't expect any filename changes in the future.)

Reply:
I just realized that it's a web/remote path (link) not a local path. You can ignore this comment!